ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/TransactionReaper.java

```diff
@@ -1020,4 +1028,4 @@ static final synchronized void reset()
     private static int _zombieCount = 0;
 
 	private boolean _inShutdown = false;
-}
\ No newline at end of file
+}
```

Comment:
A bit of formatting that could be reverted, please?

Reply:
Done, thanks Tom